includes/fs-core-functions.php

```diff
@@ -64,15 +64,33 @@ function fs_get_template( $path, &$params = null ) {
 
 	/* Scripts and styles including.
 	--------------------------------------------------------------------------------------------*/
+	function fs_asset_url( $asset_abs_path ) {
```

Comment:
@fajardoleo missing documentation. I spend 2 min to understand why is this function needed. My guess is that it's related to the fact that we have themes and plugins now, but I still don't understand why. I would recommend adding an example in the documentation to make it even clearer.